### PR TITLE
feat(bootstrap): option to scan container asset images 

### PR DESCRIPTION
**Pull request by amirfireeye**
This adds a new flag for `cdk bootstrap` called `--container-asset-scan-on-push`. It sets the generated ECR repo to be scanned on push for vulnerabilities.

We need to enable scanning but would rather not modify the bootstrap template if possible. So this flag will help us automate the process more easily by just allowing a single step for bootstrapping instead of generating template, modifying, and then deploying.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk/bin/cdk.ts [hunk 1]
```diff
@@ -79,6 +79,7 @@ async function parseCommandLineArguments() {
       .option('bootstrap-customer-key', { type: 'boolean', desc: 'Create a Customer Master Key (CMK) for the bootstrap bucket (you will be charged but can customize permissions, modern bootstrapping only)', default: undefined, conflicts: 'bootstrap-kms-key-id' })
       .option('qualifier', { type: 'string', desc: 'Unique string to distinguish multiple bootstrap stacks', default: undefined })
       .option('public-access-block-configuration', { type: 'boolean', desc: 'Block public access configuration on CDK toolkit bucket (enabled by default) ', default: undefined })
+      .option('container-asset-scan-on-push', { type: 'boolean', desc: 'Scan container asset images on push on CDK toolkit ECR (disabled by default)', default: false })
```
Comment (markusl) [inline packages/aws-cdk/bin/cdk.ts:82]: Why is this disabled by default?

Comment (amirfireeye) [reply packages/aws-cdk/bin/cdk.ts:82]: To match current behavior.

**Comment by amirfireeye** [comment]
@markusl any changes I can make to get this PR mergeable?

**Review by markusl — APPROVED** [review]

**Comment by markusl** [comment]
@amirfireeye I cannot approve the changes to be merged here. You can ask someone from the CDK team for that

File: packages/aws-cdk/bin/cdk.ts (hunk 1 above)
Comment (PatMyron) [reply packages/aws-cdk/bin/cdk.ts:82]: opened a PR to enable by default:
https://github.com/aws/aws-cdk/pull/17994

**Comment by rix0rrr** [comment]
Superseded by https://github.com/aws/aws-cdk/pull/17994

**Pull request closed without merging**